git-user-count/user-count.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash
+set -e
+
+basedir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+# input
+repositories="$basedir/repositories.txt"
```

Comment:
I think it would be a little more usable to take this as an CLI argument. It's okay if there is a default, but letting the user specify it will allow them to more easily specify different files for different uses. For example, maybe a company wants to know how many users there are by department, so they have different files for each department.